models/data_warehouse.model.lkml

```diff
@@ -3868,6 +3868,13 @@ explore: performance_events {
     fields: [server_daily_details.database_version, server_daily_details.database_version_major, server_daily_details.database_version_major_release, server_daily_details.server_version_major, server_daily_details.version, server_daily_details.edition]
   }
 
+  join: version_release_dates {
+    view_label: " Performance Events"
```

Comment:
Is this the correct label?

Reply:
I think so based on the pattern already applied

Reply:
asking because the `server_fact` below, for example, uses a different one (does not inherit the parent one)

Reply:
Honestly, I'm not really sure either way but it seems to match how `join: server_daily_details` and `join: user_agent_registry` and there's a number of other explores that use the same labels. 0/5, happy to change it if there's a better suggestion.

Reply:
yeah, the labels are incorrect for `join: server_daily_details` and `join: user_agent_registry`, @jwilander please rename the view label to 'Version Release Dates'. and leave the old ones as is, I will fix them soon.

Reply:
Done @sri-byte @catalintomai

Reply:
@sri-byte , when you do the name fixing, please open a Jira ticket for the task to keep track (also would be a good idea to do see if there are some other enhancements as part of the scrub, e.g. adding descriptions, removing unused joins, etc.) 0/5 if you want to open separate Jira tickets by improvement category.